pkg/webhook/patch_test.go

```diff
@@ -22,7 +22,7 @@ import (
 	"reflect"
 	"testing"
 
-	jsonpatch "github.com/evanphx/json-patch"
```

Comment:
Interesting, so I guess `json-patch` is automatically converted to `jsonpatch` when referenced?

Reply:
Looks like it. I'm using GoLand and it told me that this line is redundant.